### ISSUE TITLE
Description text for Matrix.org in the account provider is bogus

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-x-android#1132]
### Steps to reproduce

Change account provider during login.
Matrix.org is described as:

![image](https://github.com/vector-im/element-x-android/assets/1294269/4e7b4054-ed2e-4dbd-a0e1-da697ec09e13)

This is completely wrong - it feels like someone has taken the text from the frontpage and s/Matrix/Matrix.org/.  Obviously, Matrix.org is not the same as Matrix.

The text instead should be something like:

"Matrix.org is a large, free server on the public Matrix network for secure, decentralised communication, run by the Matrix.org Foundation" 

#### What did you expect?

Not to confuse Matrix with Matrix.org...

### Application version and app store

0.1.2 (40001020) playstore

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by jmartinesp** [comment]
This has been handled in Localazy, the strings should be updated shortly.

**Comment by bmarty** [comment]
Ideally the copy should be updated in Figma too: https://www.figma.com/file/0MMNu7cTOzLOlWb7ctTkv3/Element-X?node-id=6904%3A148203&mode=dev @amshakal can you take the point please?

**Comment by amshakal** [comment]
Sure. On it. Thanks!

**Comment by amshakal** [comment]
@americanrefugee tagging you in case the same changes need to be made on web as well

**Comment by ara4n** [comment]
thanks folks! (btw if the wording is too wordy, feel free to prune it)

**Comment by americanrefugee** [comment]
My suggestion:

"Matrix.org is a public server used for secure, decentralised communication on the Matrix network, and is run by the Matrix.org Foundation."

**Comment by jmartinesp** [comment]
This was actually modified some time ago. I'm closing it now.
